crates/ignore/src/default_types.rs

```diff
@@ -39,6 +40,7 @@ pub(crate) const DEFAULT_TYPES: &[(&[&str], &[&str])] = &[
     (&["carp"], &["*.carp"]),
     (&["cbor"], &["*.cbor"]),
     (&["ceylon"], &["*.ceylon"]),
+    (&["cfml"], &["*.[Cc][Ff][Cc]", "*.[Cc][Ff][Mm]"]),
```

Comment:
I've changed this to `*.cfc` and `*.cfm`. It doesn't make sense for everyone using this file type to pay for case insensitive matching all of the time. For users where this matters, they can use the `--glob-case-insensitive` flag.